client/__tests__/e2e/e2e.test.js

```diff
@@ -518,4 +520,20 @@ test("lasso moves after pan", async () => {
 
   expect(panCount).toBe(initialCount);
 });
+
+describe("AuthN Integration", () => {
+  if (process.env.TEST_AUTH_INTEGRATION !== "true") {
```

Comment:
if `TEST_AUTH_INTEGRATION` is not `"true"`, `it.only()` will be triggered and skip all the tests in this `describe` block